src/tools/perf/uct_tests.cc

```diff
@@ -440,6 +442,7 @@ class uct_perf_test_runner {
                     progress_responder();
                     if (UCS_CIRCULAR_COMPARE8(sn, >, (psn_t)(send_sn + (fc_window / 2)))) {
                         /* Send ACK every half-window */
+                        wait_for_window(send_window);
```

Comment:
Perhaps, `wait_for_window()` should be after `send_b()` because `m_completion.count` increments in `send_b`.